.github/workflows/src/spec-gen-sdk-status.js

```diff
@@ -177,7 +177,7 @@ async function processResult({ checkRuns, core }) {
     const shortLanguageName = language.split("-").pop();
     const executionResult = artifactJsonObj.result;
     const isSpecGenSdkCheckRequired = artifactJsonObj.isSpecGenSdkCheckRequired;
-    if (isSpecGenSdkCheckRequired && executionResult !== "succeeded") {
+    if (isSpecGenSdkCheckRequired && !(executionResult === "succeeded" || executionResult === "warning")) {
```

Comment:
Should we check for exectuionResult === "failed" instead?

Reply:
I agree since `notEnabled` state is already validated in TSV. We don't need to block here.